flux_led/__main__.py

```diff
@@ -940,7 +947,29 @@ def setRgbw(self, r=None, g=None, b=None, w=None, persist=True,
                 msg.append(int(w))
             else:
                 msg.append(int(0))
-            msg.append(special)
+
+            if self.protocol == "LEDENET":
+                # LEDENET devices support two white outputs for cold and warm. We set
+                # the second one here - if we're only setting a single white value,
+                # we set the second output to be the same as the first
+                if w2 is not None:
```

Comment:
https://github.com/Danielhiversen/flux_led/blob/47c6e02e178dae19c871c9e1bde26ae035cd5213/flux_led/__main__.py is the commit just before the LEDENET_ORIGINAL was added. If you look at how rgbwprotocol is used, you'll see that they're 8-byte protocols that don't use the write mask. This only applies to types 0x04, 0x33 and 0x81.